docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,61 @@
+# Relative Layout
+
+`RelativeLayout` is a layout class in which each side of children can be specified as relative to sibling view or parent. <br/>
+`Target` and `RelativeOffset` are used to define relationship to each other.
+ 
+A relative layout is useful to keep layout hierarchy flat. <br/>
+For example, a view using two linear layout is able to replace with a single relative layout.
+| Linear layout | Relative layout |
+| ------------- | --------------- |
+|![relativeLayoutLinearExample1](./media/relativeLayoutLinearExample1.png)|![relativeLayoutLinearExample2](./media/relativeLayoutLinearExample2.png)|
+
+Here are the attached properties of `RelativeLayout`:
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `LeftTarget`           | View            | Gets or sets the left side edge of the child view relative to the given target view.|
+| `RightTarget`          | View            | Gets or sets the right side edge of the child view relative to the given target view.|
+| `TopTarget`            | View            | Gets or sets the top side edge of the child view relative to the given target view.|
+| `BottomTarget`         | View            | Gets or sets the bottom side edge of the child view relative to the given target view.|
+| `LeftRelativeOffset`   | float           | Gets or sets the relative offset for left target. <br /> When the value is `0` the left edges of the left target and child view are aligned.<br/> When the value is `1` the left edge of the child view is aligned to the right edge of the left target. |
+| `RightRelativeOffset`  | float           | Gets or sets the relative offset for right target. <br /> When the value is `0` the right edge of the child view is aligned to the left edge of the right target.<br/> When the value is `1` the right edges of the right target and child view are aligned.
+| `TopRelativeOffset`    | float           | Gets or sets the relative offset for top target. <br /> When the value is `0` the top edges of the top target and child view are aligned.<br/> When the value is `1` the top edge of the child view is aligned to the bottom edge of the top target.
+| `BottomRelativeOffset` | float           | Gets or sets the relative offset for bottom target. <br /> When the value is `0` the bottom edge of the child view is aligned to the top edge of the right target.<br/>When the value is `1` the bottom edges of the bottom target and child view are aligned.
+| `HorizontalAlignment`  | RelativeLayout.Alignment| Gets or sets the horizontal alignment of this child view.|
+| `VerticalAlignment`  | RelativeLayout.Alignment| Gets or sets the vertical alignment of this child view.|
+| `FillHorizontal`  | bool | Gets or sets the boolean value whether the child fills its horizontal space.|
+| `FillVertical`  | bool | Gets or sets the boolean value whether the child fills its vertical space.|
```

Comment:
Description needs rephrase, I have tried as per my understanding, please check for its correctness. 
```suggestion 
| `FillHorizontal`  | bool | Gets or sets the boolean value when the child fills its horizontal space.|
| `FillVertical`  | bool | Gets or sets the boolean value when the child fills its vertical space.|

Reply:
What do you think of 
```suggestion
| `FillHorizontal`  | bool | Gets or sets the boolean value indicating whether the child fills its horizontal space or not .|
| `FillVertical`  | bool | Gets or sets the boolean value indicating whether the child fills its vertical space or not.|
```